kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -302,6 +302,7 @@ export class FormWizard extends HTMLElement {
       let name = step.getAttribute("name");
       if (name === activeStep) {
         step.slot = "active";
+        step.activate && step.activate();
       } else {
         step.slot = "";
```

Comment:
I like this approach! But I'm going to suggest we switch this around, because that's going to work better with the patch I'm working on for only setting focus to the email field in particular circumstances.

Instead of having an `activate`, can we have a deactivate? And have the deactivate on the sign-in-step be something like this:

```js
    if (this.shadowRoot.activeElement == this.#emailEl) {
      this.#emailEl.blur();
      this.#emailErrorEl.classList.remove("visible");
    }
```

Reply:
Also, maybe instead of checking for the presence of a `deactivate` method, can we make it a NOOP method on the base class that subclasses can override?

Reply:
this works well thanks for the suggestions! calling `blur` will make it so that the input is no longer focused when the user gets sent back to step 1, does that seem like an issue? I feel like if the step changes it should be pretty obvious that the form is interactive...

Reply:
I agree. I think the only autofocus case we truly care about at this point is upon visiting the page and hitting Step 1.